LICENSE.Apache

```diff
@@ -178,15 +178,16 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "{}"
```

Comment:
Huh, any idea why this change was made? Could there be a purpose we're not aware of for this `LICENSE.Apache` file?

Reply:
There are 2 versions of the Apache license one use `{}` the other one uses `[]`, I replaced all Apache licenses with the same file with `[]` that's why this change was made